test/unit/spec/common/validate2-Spec.js

```diff
@@ -1,6 +1,6 @@
 define([
     'bluebird',
-    'common/validation',
+    'widgets/appWidgets2/validation',
```

Comment:
Updated this one (why `validate2`? Why not?) to use the merged file.
Will probably go away in the next PR.